tools/distpartitioning/utils.py

```diff
@@ -1,18 +1,19 @@
 import json
 import logging
 import os
+from contextlib import contextmanager
```

Comment:
why import this here? Is it used directly in this file?

Reply:
Deleted